### ISSUE TITLE
[Bug] error when adding decorators to g.gui.animation

### ISSUE THREAD

**Issue by veroandreo** [OSGeo/grass#834]
When trying to add decorators like time stamps or text to an animation, I get:

![image](https://user-images.githubusercontent.com/20075188/88465341-81825100-cec2-11ea-8eff-6af58c2a59f8.png)

**To Reproduce**
1. Create an animation
2. Go to export
3. Add time stamps and hit `Export`

**System description:**
```
GRASS version: 7.8.3                                                            
Code revision: exported                                                         
Build date: 2020-06-26                                                          
Build platform: x86_64-redhat-linux-gnu                                         
GDAL: 3.0.4                                                                     
PROJ: 6.3.2                                                                     
GEOS: 3.8.1                                                                     
SQLite: 3.32.3                                                                  
Python: 3.8.3                                                                   
wxPython: 4.0.7                                                                 
Platform: Linux-5.7.8-200.fc32.x86_64-x86_64-with-glibc2.2.5   
```

I will be teaching different GRASS tools starting on August 3rd and for 3 weeks, and it's a pity that so many GUI tools are broken in the stable version.

**Comment by petrasovaa** [comment]
You didn't actually select the method, so although this is still a bug, it is at least easy to avoid it.

**Comment by veroandreo** [comment]
@petrasovaa Right! That was my bad... sorry for the noise

However, when I try to export to gif, ie. first select gif, browse for directories and set a name with extension (bla.gif), I get the error reported in #827 

I close this one as it is invalid

**Comment by petrasovaa** [comment]
Not quite invalid, anyway, should be solved by #838.
